introduction.md

```diff
@@ -1 +1,5 @@
-Hello Dojo, let me introduce myself. I'm Devan and my favorite Code School path is Javascript.
\ No newline at end of file
+Hello Dojo, let me introduce myself. I'm Devan and my favorite Code School path is Javascript.
+Deadly Skills
+*Harry Potter
+*Silly Putty
+*Fall Out Boy
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?